tests/config/bug_1002.cfg

```diff
@@ -0,0 +1,6 @@
+input_tab_size                  = 4
+output_tab_size                 = 4
+indent_columns                  = 4
+sp_arith                        = force
+sp_assign                       = force
+sp_bool                         = force
```

Comment:
Are all of those needed? 
As far as I understand the problem only `sp_before_byref`/`sp_after_byref` and `sp_arith` should be involved. 
The option `sp_arith` should be activated while both of the other options should not.